<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginUserClass</class>
 <widget class="QDialog" name="LoginUserClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login User</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpLogin">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Login</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblLanguages">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string>Languages </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QComboBox" name="cmbLanguages">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>19</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>عربي</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblUserName">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string>User Name            </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2" colspan="5">
       <widget class="QLineEdit" name="txtUserName">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>19</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLabel" name="lblUserPassword">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>19</height>
         </size>
        </property>
        <property name="text">
         <string>Password              </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QLineEdit" name="txtUserPassword">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>19</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>16</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>243</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="chkSaveUser">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnLogin">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="btnCancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QToolButton" name="btnMore">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblLocation">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>19</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbLocation">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>19</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblRoom">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>19</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>88</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Room</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbRoom">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>19</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnMore</sender>
   <signal>clicked(bool)</signal>
   <receiver>frame</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
